website/pages/docs/job-specification/gateway.mdx

```diff
@@ -199,12 +199,30 @@ make use of the envoy version interpolation, e.g.
 meta.connect.gateway_image = custom/envoy-${NOMAD_envoy_version}:latest
 ```
 
+### Custom gateway task
+
+The task created for the gateway can be configured manually using the
+[`sidecar_task`][sidecar_task] stanza.
```

Comment:
The [`connect` parameters](https://www.nomadproject.io/docs/job-specification/connect#connect-parameters) docs don't currently have any reference to the gateway. I feel like we need something explaining what happens if the `sidecar_task` is set if both `sidecar_service` and `gateway` blocks are used (and whether that's allowed at all!).

Reply:
Oh wow yea, it's totally missing `gateway`. Will add.